pyInterface/storageFormats/amplitudeFileWriter_py.cc

```diff
@@ -0,0 +1,69 @@
+
+#include "amplitudeFileWriter_py.h"
+
+#include "rootConverters_py.h"
+#include "stlContainers_py.h"
+
+namespace bp = boost::python;
+
+
+namespace {
+
+	bool amplitudeFileWriter_initialize(rpwa::amplitudeFileWriter& self,
+	                                    PyObject*                  pyOutputFile,
+                                        bp::object                 pyEventMetadata,
+                                        const std::string&         keyfileContent,
+                                        const std::string&         objectBasename,
+                                        const int&                 splitlevel = 99,
+                                        const int&                 buffsize = 256000)
```

Comment:
indentation errors in line 14-18, use spaces instead of tabs for everything but the first tab.